src/NSwag.CodeGeneration.CSharp/CSharpControllerGeneratorSettings.cs

```diff
@@ -20,6 +20,7 @@ public CSharpControllerGeneratorSettings()
         {
             ClassName = "{controller}";
             CSharpGeneratorSettings.ArrayType = "System.Collections.Generic.List";
+            CSharpGeneratorSettings.ArrayInstanceType = "System.Collections.Generic.List",
```

Comment:
Inst the fallback already the value of ArrayType?

Reply:
Nope, according to
https://github.com/RicoSuter/NJsonSchema/blob/96d37dc3abf1c154aaffb9dab2ca8f5c1f5bbdc9/src/NJsonSchema.CodeGeneration.CSharp/CSharpGeneratorSettings.cs#L28
We need to change it here too